.github/workflows/build.yml

```diff
@@ -43,5 +43,9 @@ jobs:
           image: ${{ inputs.image }}
           context: ${{ inputs.context }}
           stage: ${{ inputs.stage }}
+          tags: |
```

Comment:
Here is the doc for this
https://github.com/marketplace/actions/docker-metadata-action#typeref